examples/xgboost/run_experiment_simulator.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+n=5
+for study in bagging_uniform_split_uniform_lr bagging_exponential_split_uniform_lr bagging_exponential_split_scaled_lr cyclic_uniform_split_uniform_lr cyclic_exponential_split_uniform_lr
+do
+  nvflare simulator job_configs/higgs_${n}_${study} -w ${PWD}/workspaces/xgboost_workspace_${n}_${study} -n ${n} -t ${n}
+done
+
+
+n=20
+for study in bagging_uniform_split_uniform_lr bagging_square_split_uniform_lr bagging_square_split_scaled_lr cyclic_uniform_split_uniform_lr cyclic_square_split_uniform_lr
+do
+  nvflare simulator job_configs/higgs_${n}_${study} -w ${PWD}/workspaces/xgboost_workspace_${n}_${study} -n ${n} -t ${n}
+done
```

Comment:
This is for bagging only, maybe add a line for the histogram exp?

Reply:
sure, I separate this for each setup.